config.json

```diff
@@ -1,585 +1,588 @@
-{
```

Comment:
Please revert all changes to this file besides the addition of the two new RPCs and the removal of `OnButtonSubscription` notification